kolibri/utils/options.py

```diff
@@ -32,6 +32,28 @@
             "envvars": ("KOLIBRI_DATABASE_HOST",),
         },
     },
+    "Server": {
+        "SERVER_THREAD_POOL": {
```

Comment:
Not sure if a previous comment of mine has been lost during force-pushing or never made it. Anyways, I would suggest renaming `SERVER_THREAD_POOL` to `CHERRYPY_THREAD_POOL` and the envvar to `KOLIBRI_CHERRYPY_THREAD_POOL`.

The same thing for the other variables.

The reason is that they don't apply to the whole server, and we might have alternatives to CherryPy, disable it etc.